spaces/domains.json

```
@@ -1,4 +1,5 @@
 {
+  "vote.dsbswap.com":"dsbswap.eth",
```

Please add a space here.
```suggestion
  "vote.dsbswap.com": "dsbswap.eth",
```